src/cmd/create.go

```diff
@@ -347,14 +346,13 @@ func createContainer(container, image, release string, showCommandToEnter bool)
 		"--network", "host",
 		"--no-hosts",
 		"--pid", "host",
-		"--privileged",
 		"--security-opt", "label=disable",
 	}
 
 	createArgs = append(createArgs, ulimitHost...)
 
 	createArgs = append(createArgs, []string{
-		"--userns=keep-id",
+		// "--userns=keep-id",
```

Comment:
This cannot be right. :)

Wasn't there a bug in Podman that broke `--userns=keep-id` for rootless containers that later got fixed?

Reply:
Yes, exactly. I was playing with this and pushed it with it being commented. It should be uncommented, of course.

Reply:
For those following along at home, I was referring to https://github.com/containers/libpod/issues/6652